documentation/asciidoc/stories/assembly_connecting_clients.adoc

```diff
@@ -0,0 +1,30 @@
+ifdef::context[:parent-context: {context}]
+[id='connecting_clients']
+:context: clients
+= Establishing Remote Client Connections
+Connect to {brandname} clusters from the {brandname} Console, Command Line Interface (CLI), and remote clients.
+Remote client connections let you create caches from valid configuration in XML or JSON format, operate on data, and perform administrative operations such as retrieving statistics and monitoring cluster health.
```

Comment:
How about simplifying to just:

"Remote client connections allow you to connect to {brandname} clusters from the {brandname} Console, Command Line Interface (CLI), and remote clients."